plugins/woocommerce/client/blocks/assets/js/blocks/checkout/frontend.tsx

```diff
@@ -56,11 +56,15 @@ const Wrapper = ( {
 renderParentBlock( {
 	Block,
 	blockName,
-	selector: '.wp-block-woocommerce-checkout',
+	selector:
+		'.wp-block-woocommerce-checkout[data-block-name="woocommerce/checkout"]',
 	getProps,
 	blockMap: getRegisteredBlockComponents( blockName ) as Record<
 		string,
 		React.ReactNode
 	>,
 	blockWrapper: Wrapper,
+	options: {
+		multiple: false,
```

Comment:
is it possible to get this from the metadata in block.json instead of defining it here? Ideally, this logic would apply to all of our blocks where `multiple: false` is set not just the checkout.

Reply:
The usage of `renderParentFrontend` throughout the project is minimal (4 places). I would prefer not to complicate it more by passing a pretty extensive configuration object that will be mostly unused. As an alternative/compromise, I propose:
- to make `options` required parameter (for it to be enforced we would have to fix some TS issues that we have with the current usage of this function, as other errors are preventing from checking the presence of all required arguments)
- not provide a literal `multiple: false` for options but import the value from existing metadata

The above proposition is pushed in https://github.com/woocommerce/woocommerce/pull/58001/commits/97f9dcafd7744ec1c4a102effca20c10c1fc11db so you can inspect it in practice.